packages/mobile/src/consumerIncentives/ConsumerIncentivesHomeScreen.tsx

```diff
@@ -193,7 +202,7 @@ function ClaimSuperchargeRewards({ rewards }: { rewards: SuperchargePendingRewar
   )
 }
 
-function useFetchAvailableRewards(): {
+export function useFetchAvailableRewards(): {
```

Comment:
Note for later: If we start sharing those fetch calls, we should consider moving them to Redux / Redux Toolkit Query.
Main reason:
> - Avoiding duplicate requests for the same data
> - Optimistic updates to make the UI feel faster
> - Managing cache lifetimes as the user interacts with the UI

https://redux-toolkit.js.org/rtk-query/overview

Reply:
Oh, I didn't know about RKT Query. I was purposefully querying again to have up to date data in this screen because I didn't want to write custom cache and cache invalidation logic, but looks like that solves it.

On the other hand, I've been using `createAsyncThunk` on Celo Tracker and like it a lot more than `redux-saga` (I never liked sagas from the beginning though to be fair). Do you think it would make sense to try to use them in some reducer or would we prefer to maintain consistency with out current approach?

Reply:
I think it's great we experiment with new approaches! 

We used to have thunk until we decided to only use `redux-saga` for consistency. I agree `redux-saga` can be complex and difficult. There's really a gap between thunk and saga, thunk is great until more complex logic is needed which requires listening to one multiple dispatched actions.

I saw something quite promising, Redux Toolkit is experimenting with a new API to cover just that, alleviating the need for sagas: https://twitter.com/acemarke/status/1498040623487758339
I think we should take a look at this.